# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../clockspeed/0001-make-makelib.sh.patch
# Copyright (C) 2011 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

From 41ab756f8c4c2216b1e9a53bee04ad6749558ce4 Mon Sep 17 00:00:00 2001
From: Christian Wiese <christian.wiese@securepoint.de>
Date: Thu, 29 Sep 2011 14:11:40 +0200
Subject: [PATCH 1/6] make-makelib.sh: fix to not hardcode 'ar' and 'ranlib'

---
 make-makelib.sh |    4 ++--
 1 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/make-makelib.sh b/make-makelib.sh
index d6b7c8c..ea0a6ab 100644
--- a/make-makelib.sh
+++ b/make-makelib.sh
@@ -1,6 +1,6 @@
 echo 'main="$1"; shift'
 echo 'rm -f "$main"'
-echo 'ar cr "$main" ${1+"$@"}'
+echo '${AR:-ar} cr "$main" ${1+"$@"}'
 
 case "$1" in
 sunos-5.*) ;;
@@ -11,6 +11,6 @@ dgux-*) ;;
 hp-ux-*) ;;
 sco*) ;;
 *)
-  echo 'ranlib "$main"'
+  echo '${RANLIB:-ranlib} "$main"'
   ;;
 esac
-- 
1.6.6.2

